integration/service/src/main/resources/com/cerner/beadledom/integration/service/build-info.properties

```diff
@@ -0,0 +1,4 @@
+git.commit.id=${git.commit.id}
```

Comment:
This is for my knowledge, why do we need this file?

Reply:
This file is used by the ServiceMetadata to help swagger get some metadata about the service. It's used here: https://github.com/cerner/beadledom/pull/149/files/070df335fae218738ee9473d9c6aa2abcc2c0114#diff-fad1ad5b719cd2aebc6b4b10bdcfc825R34

Reply:
Come to think of it, we only really need the project version out of this file, so we might be able to remove all the other key value pairs....